skycatalogs/catalog_creator.py

```diff
@@ -1165,5 +1193,6 @@ def write_provenance_file(self, datafile_path):
         '''
         outpath = datafile_path.rsplit('.', 1)[0] + '_provenance.yaml'
 
-        prov = assemble_provenance(self._pkg_root, inputs=None)
+        prov = assemble_provenance(self._pkg_root, inputs=None,
```

Comment:
It seems inconsistent that we have `inputs=None` for this provenance and an `inputs` value that's not `None` above.  Shouldn't the same provenance info be used in both places?

Reply:
Not necessarily.  In the first case, a global config file is being written. The value for `inputs` is all the possible inputs catalog_creator knows about.  In the second, it's only associated with a particular binary file.  Probably there should have been an `inputs` argument to write_provenance_file so that inputs appropriate to the object type of that file could be included.   But, now that provenance is being included in the binary file, I don't think we need this routine altogether.  My inclination is to get rid of it and the option in the creation script which causes it to be called.